hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/multipart/S3MultipartUploadAbortRequest.java

```diff
@@ -120,6 +124,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
         multipartKeyInfo = omMetadataManager
             .getMultipartInfoTable().get(multipartKey);
 
+        multipartKeyInfo.setUpdateID(transactionLogIndex);
```

Comment:
Same as above, I think we don't need to set updateID here, as anyway we are removing entry from multipartInfo table. Is there is any use case that will help in the detection of replay by setting here?